src/main/java/com/gregtechceu/gtceu/common/network/GTNetwork.java

```diff
@@ -31,35 +32,50 @@ public class GTNetwork {
     private static final SimpleChannel INSTANCE = NetworkRegistry.newSimpleChannel(GTCEu.id("network"),
             () -> PROTOCOL_VERSION, PROTOCOL_VERSION::equals, PROTOCOL_VERSION::equals);
 
+
+    private static void protectedSend(PacketDistributor.PacketTarget target, INetPacket packet) {
+        try {
+            INSTANCE.send(target, packet);
+        } catch (Exception e) {
+            GTCEu.LOGGER.warn("Failed to send packet: {}", e.getLocalizedMessage());
+        }
+    }
+
     private static int nextPacketId = 0;
 
     public static void sendToServer(INetPacket packet) {
-        INSTANCE.sendToServer(packet);
+        try {
+            INSTANCE.sendToServer(packet);
+        } catch (Exception e) {
+            // If the send failed because the server connection was lost, ignore the error.
+            if (Minecraft.getInstance().getConnection() == null) return;
```

Comment:
should we try to detect that it specifically was a disconnection that cause it to fail?

Reply:
also spotless